modules/RemoveBook.js

```diff
@@ -0,0 +1,9 @@
+import bookCollection from './BookCollection.js';
+import displayBooks from './BookManager.js';
+
+function removeBook(index) {
```

Comment:
- [ ] As using ES6 syntax is a requirement in this milestone, kindly consider using ES6 arrow functions instead of random function keyword.